arch/xtensa/core/startup/reset-vector.S

```diff
@@ -24,8 +24,8 @@
 	.section		.ResetVector.text, "ax"
 
 	.align	4
-	.global	_ResetVector
-_ResetVector:
+	.global	__start
+__start:
```

Comment:
Well.. on S1000, the "make debug" and "make flash" commands download the ELF file on to the beginning of SRAM (address is 0xBE000000). Upon Power on, ROM code gets executed and jumps to SRAM location 0xBE000000. So, the first piece of code that we want to be executed in SRAM needs to be placed at 0xBE000000. Since there is already a different __start in the Xtensa code, that was getting placed at that location thereby causing issues. Hence _ResetVector had to be renamed to __start to ensure it gets included in the right area.